### ISSUE TITLE
uCI Release: Go v1.20.0

### ISSUE THREAD

**Issue by galargh** [protocol/.github#453]
#### Meta

_Expected date_: 2023-02-06

###### Required

- [ ] https://github.com/protocol/.github/pull/443
- [ ] https://github.com/protocol/.github/pull/280

###### Nice to have

- [ ] https://github.com/protocol/.github/pull/299
- [ ] https://github.com/protocol/.github/pull/295

**Comment by marten-seemann** [comment]
It looks like `staticcheck` is failing with Go 1.20: https://github.com/libp2p/go-libp2p/actions/runs/4079757526/jobs/7031459314

This is probably related to https://github.com/dominikh/go-tools/issues/1358.

I suggest blocking the Unified CI release on the staticcheck update.

**Comment by dominikh** [comment]
FYI, the release will likely happen within the next 1-2 hours.

**Comment by marten-seemann** [comment]
> FYI, the release will likely happen within the next 1-2 hours.

Great, thank you for your excellent work on staticcheck, @dominikh!

**Comment by marten-seemann** [comment]
It's here: https://github.com/dominikh/go-tools/releases/tag/2023.1 🎉

**Comment by galargh** [comment]
Thank you both ❤️❤️❤️❤️❤️❤️❤️❤️❤️❤️❤️

**Comment by marten-seemann** [comment]
> Expected date: 2023-02-06

Any update on the timeline?

**Comment by galargh** [comment]
@marten-seemann I wanted your opinion on https://github.com/protocol/.github/pull/463 because it'd be good to include automated fix for rand.Seed deprecation. 

Once that's merged, we can proceed. So hopefully either today or tomorrow.

**Comment by marten-seemann** [comment]
Oh sorry, I missed that PR. Just gave it a review.

**Comment by galargh** [comment]
Thanks for the discussion there :) We'll go ahead with the release today.

Deploying to the repos on `source_ref: next` now to verify everything is in order.

**Comment by galargh** [comment]
Deploy to the repos with `source_ref: next` went through as expected. 2 PRs were automatically merged and the other 2 were held up by Go Check workflow on deprecation notices. 

Creating a PR to master now.

**Comment by galargh** [comment]
The release PR is ready at https://github.com/protocol/.github/pull/466. Waiting for checks to finish and browsing through changes before merging.

**Comment by galargh** [comment]
Review complete - https://github.com/protocol/.github/pull/466/commits/cbbd6f0b42557972799e6abffcb093768266a2ad - and tested on the `testing` branch - https://github.com/protocol/.github/commit/0b2ba34417d0ffc8d9b29fab684a3b01be038b2d.

Proceeding with the release.

**Comment by galargh** [comment]
The release's complete 🥳
